docs/markdown/Release-notes-for-0.48.0.md

```diff
@@ -310,7 +310,7 @@ files get exported.
 To enable this, the following needs to be added to the `.wrap` file:
 
 ```ini
-clone-recursive=true
+clone-recursive = true
```

Comment:
For consistency, I added a space on both sides of the = sign (to match the examples on the wrap manual page).